### ISSUE TITLE
0.5.0 version crushes PhantomJS

### ISSUE THREAD

**Issue by brave44** [jedisct1/libsodium.js#85]
After libsodium.js was updated to 0.5.0, Karma+PhantomJS returns error. Everything works fine with 0.4.9 version, but I need 0.5.0 because of kx_ functions. We think may be duplicated object properties cause that error in phantomjs, but we not sure.

`24 03 2017 07:38:16.739:INFO [karma]: Karma v1.5.0 server started at http://0.0.0.0:9876/
24 03 2017 07:38:16.741:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
24 03 2017 07:38:16.824:INFO [launcher]: Starting browser PhantomJS
24 03 2017 07:38:17.675:INFO [PhantomJS 2.1.1 (Linux 0.0.0)]: Connected on socket dyBbdQkD2bKalzfLAAAA with id 87678580
PhantomJS 2.1.1 (Linux 0.0.0) ERROR
  SyntaxError: Unexpected token ','
  at webpack:///home/vagrant/~/libsodium/dist/modules/libsodium.js:21:0 <- karma.context.js:44996
`

**Comment by artemeff** [comment]
Yes, crashes due to keys duplication, at the end of the object at line 21 started from `stackAlloc`:

```
... d_chacha20poly1305_ietf_npubbytes:Pd,runPostSets:zc,stackAlloc:vd,stackSave:ge,stackRestore:Rd,establishStackSpace:Jd,setTempRet0:Sd,getTempRet0:ce,setThrew:Gd,stackAlloc:vd,stackSave:ge,stackRestore:Rd,establishStackSpace:Jd,setThrew:Gd,setTempRet0:Sd,getTempRet0:ce}})
                                                         ^ here                                                                                                      ^ and here
```

If you delete duplicates, it works. But I don't know why it even crashes 😑

**Comment by jedisct1** [comment]
Can you try removing the `use strict` statement?

**Comment by brave44** [comment]
@jedisct1 yeah, it helped. Now what? ))
